leetcode/994_rotting_oranges.py

```diff
@@ -0,0 +1,35 @@
+class Solution:
+    def orangesRotting(self, grid: List[List[int]]) -> int:
+
+        res = 0
+        queue = deque()
+
+        for i in range(len(grid)):
+            for j in range(len(grid[0])):
+                if grid[i][j] == 2:
+                    queue.append(((i,j),0))
```

Comment:
When appending to the queue during the first traversal, directly append (i, j, 0) instead of wrapping (i, j) in a tuple. This will make unpacking in the BFS loop cleaner and easier to read.

```suggestion
                    # Add initial rotten orange coordinates with time
                    queue.append((i, j, 0))  
```